treeFormatting.js

```diff
@@ -5,10 +5,10 @@ var categoryBrackets = {
 	"xp": "[]",
 	"phi": "()",
 	"x0": ["[x0 ","]"],
-	"w": ["(w ", ")"],
+	"w": ["[", "]"],
 	"clitic": ["",""],
 	"syll": ["",""],
```

Comment:
Change to ["(s", ")"], in case we ever need to use syll as an intermediate category.